wire-ios-data-model/Tests/MLS/MLSServiceTests.swift

```diff
@@ -44,11 +44,12 @@ class MLSServiceTests: ZMConversationTestsBase, MLSServiceDelegate {
     let groupID = MLSGroupID([1, 2, 3])
 
     // `MLSService.init` performs actions which make it hard to assert state on mocks
-    let skipSetupSUTTestNames: Set<String> = [
-        "-[MLSServiceTests test_BackendPublicKeysAreFetched_WhenInitializing]",
-        "-[MLSServiceTests test_UpdateKeyMaterial_WhenInitializing]",
-        "-[MLSServiceTests test_SendPendingProposal_BeforeUpdatingKeyMaterial_WhenInitializing]"
+    let skipSetupSUTTestNames = Set([
+        "-[MLSServiceTests \(#selector(test_BackendPublicKeysAreFetched_WhenInitializing))]",
+        "-[MLSServiceTests \(#selector(test_CreateGroup_IsSuccessful))]"
     ]
+        .map { $0.replacingOccurrences(of: "WithCompletionHandler:", with: "") }
```

Comment:
do we need this?

Reply:
The issue I tried to solve was that method names had been changed without updating the list.
My solution was to add `#selector` because the compiler makes sure, that the mentioned method exists.
However, the resulting string contains the Cbjective-C name of the method while `self.name` lists the name as it is defined in Swift.